src/Bridge/Doctrine/Orm/ItemDataProvider.php

```diff
@@ -94,13 +102,21 @@ public function getItem(string $resourceClass, $id, string $operationName = null
 
         $repository = $manager->getRepository($resourceClass);
         $queryBuilder = $repository->createQueryBuilder('o');
-
+        $i = 0;
         foreach ($identifiers as $propertyName => $value) {
             $placeholder = 'id_'.$propertyName;
+            $whereExpr[$i] = $queryBuilder->expr()->eq('o.'.$propertyName, ':'.$placeholder);
+
+            if ($i <= 0) {
```

Comment:
```
 foreach ($identifiers as $propertyName => $value) {
            $placeholder = 'id_'.$propertyName;
            $whereExpr[] = $queryBuilder->expr()->eq('o.'.$propertyName, ':'.$placeholder);
            $whereExprObj = new \ArrayObject($whereExpr);
            var_dump($whereExprObj);
            $whereIterator = $whereExprObj->getIterator();
            var_dump($whereIterator->key());
            if ($whereIterator->key() <= 0) {
                $queryBuilder
                    ->where($whereIterator->current());
            } else {
                $queryBuilder
                    ->andWhere($whereIterator->current());
            }
            $queryBuilder->setParameter($placeholder, $value);

            $whereIterator->next();
        }
```

I can't make it working since the array is populated at the same time that I test it keys.
Do you see something wrong in what i've tried to achieve ?